pkg/scheduler/api/node_info.go

```diff
@@ -153,7 +153,7 @@ func NewNodeInfo(node *v1.Node) *NodeInfo {
 		nodeInfo.Allocatable = NewResource(node.Status.Allocatable).Add(nodeInfo.OversubscriptionResource)
 		nodeInfo.Capability = NewResource(node.Status.Capacity).Add(nodeInfo.OversubscriptionResource)
 	}
-	nodeInfo.setNodeGPUInfo(node)
+	nodeInfo.SharedDevices[GPUSharingDevice] = gpushare.NewGPUDevices(nodeInfo.Name, node)
```

Comment:
Suggest to use `setNodeOthersResource` to encapsulation the `SharedDevices[GPUSharingDevice] = gpushare.NewGPUDevices(nodeInfo.Name, node)`